packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -78,6 +79,7 @@ import {
   ImagePlaneModule,
 } from '../types';
 import ViewportStatus from '../enums/ViewportStatus';
+import { Enums } from '..';
```

Comment:
can you directly import the enum that you need? This often results in circular dependencies